src/routes/write-exercise/WriteExerciseScreen.tsx

```diff
@@ -56,38 +58,41 @@ const WriteExerciseScreen = ({ navigation, route }: WriteExerciseScreenPropsType
   }
 
   const docsLength = documents?.length ?? 0
+  const document = documents?.[currentDocumentNumber]
 
   return (
-    <View>
+    <>
       <ExerciseHeader
         navigation={navigation}
         route={route}
         currentWord={currentDocumentNumber}
         numberOfWords={docsLength}
       />
 
-      {documents !== null && documents[currentDocumentNumber]?.document_image.length > 0 && (
-        <>
-          <KeyboardAwareScrollView
-            scrollEnabled={false}
-            resetScrollToCoords={{ x: 0, y: 0 }}
-            enableOnAndroid
-            keyboardShouldPersistTaps='always'>
-            {!newDocuments && <Spinner />}
-            <ImageCarousel images={documents[currentDocumentNumber]?.document_image} />
-            <AnswerSection
-              currentDocumentNumber={currentDocumentNumber}
-              setCurrentDocumentNumber={setCurrentDocumentNumber}
-              documents={documents}
-              finishExercise={finishExercise}
-              tryLater={tryLater}
-              trainingSet={trainingSet}
-              disciplineTitle={disciplineTitle}
-            />
-          </KeyboardAwareScrollView>
-        </>
+      {documents && document ? (
+        <ScrollView
```

Comment:
I actually like that the content is moved to the top and the image is only displayed partly if opening the keyboard as more of the buttons are visible which makes it easier for users to solve the exercise imo.

I fixed the text cut off issue though, wanted to do it in a separate issue because I struggled with it but all good now I hope. I also reduced the margins a little bit so that more content can be shown.

Won't do the keyboard here if there is a separate issue/was there before.